xray/audit/jas/applicabilitymanager.go

```diff
@@ -156,7 +156,7 @@ func (a *ApplicabilityScanManager) createConfigFile(workingDir string) error {
 // Runs the analyzerManager app and returns a boolean to indicate whether the user is entitled for
 // advance security feature
 func (a *ApplicabilityScanManager) runAnalyzerManager() error {
-	return a.scanner.analyzerManager.Exec(a.scanner.configFileName, applicabilityScanCommand, a.scanner.serverDetails)
+	return a.scanner.analyzerManager.Exec(a.scanner.configFileName, applicabilityScanCommand, a.scanner.analyzerManager.GetAnalyzerManagerDir(), a.scanner.serverDetails)
```

Comment:
can't we use `a.scanner.analyzerManager.GetAnalyzerManagerDir()` from inside?

Reply:
no, this can be different for types of scanners.
For CA, Secrets and IAC - the working dir should be the AnalyzerManager dir
for SAST - the working dir to activate the AnalyzerManager binary on should be the actual working directory we want to scan.
 the method is Exec of analyzerManager and not the scanner, so we provide it like that